src/patternfly/components/Dropdown/dropdown.hbs

```diff
@@ -1,20 +1,20 @@
-<div class="pf-c-dropdown{{#if pf-c-dropdown--IsExpanded}} pf-m-expanded{{/if}} {{pf-c-dropdown--Modifier}}">
+<div class="pf-c-dropdown{{#if dropdown--IsExpanded}} pf-m-expanded{{/if}} {{dropdown--modifier}}">
     <button id="{{id}}"
```

Comment:
is it possible to use a `{{> button }}` here instead of HTML?

Reply:
fixed